libs/components/data-manager/src/lib/modules/data-manager/models/data-manager-state.ts

```diff
@@ -4,13 +4,39 @@ import { SkyDataManagerStateOptions } from './data-manager-state-options';
 import { SkyDataViewState } from './data-view-state';
 import { SkyDataViewStateOptions } from './data-view-state-options';
 
+/**
+ * Provides options for dictating what data is displayed.
```

Comment:
```suggestion
 * Provides options that control which data to display.
```
Ignore if this doesn't work, but something about `for dictating` felt off to me.